scripts/cnode-helper-scripts/deploy-dbsync-functions.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
+# shellcheck disable=SC1090,SC2086
+
+PARENT="$(dirname "$0")" 
+. "${PARENT}"/env offline
+
+if ! command -v psql &>/dev/null; then 
+  echo -e "${FG_RED}ERROR${NC}: psql command not found, make sure that you have Cardano DBSync setup correctly"
+  echo -e "\nhttps://cardano-community.github.io/guild-operators/#/Build/dbsync\n"
```

Comment:
Should we point this to https://cardano-community.github.io/guild-operators/#/Appendix/postgres ?

Reply:
👍